spec/syntax.md

```diff
@@ -732,10 +731,11 @@ _Function_ _identifiers_ are prefixed with `:`, `+`, or `-`.
 _Option_ _identifiers_ have no prefix.
 
 A **_<dfn>name</dfn>_** is a character sequence used in an _identifier_ 
-or as the name for for a _variable_.
+or as the name for for a _variable_
+or the value of an _unquoted_ _literal_.
 
 _Variable_ names are prefixed with `$`.
-for a _function_ (prefixed with `:`, `+` or `-`),
+for a _function_ (prefixed with `:`, `+` or `-`).
```

Comment:
This line is missing the rest of itself. What's this trying to say?

Reply:
It was missing when I got here. Fixed this to be consistent with the previous sentence.